shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -351,7 +352,7 @@ export default {
         return !this.value.mgmt?.isReady;
       }
 
-      if ( this.value.isCustom ) {
+      if ( this.value.isCustom || this.value.isAKS || this.value.isEKS ) {
```

Comment:
1) Should these two additions take in to account the ready state of the cluster as per line 352
2) Will this change mean that the registration tab will show up for clusters manually created in the dashboard (and not imported), is that an issue?

Reply:
I've never really dived in to creating AKS/EKS via the dashboard, but my assumption was that it would work the same as creating RKE1/2 and that adding new nodes would be done via editing the cluster rather than manually running a registration command (so the registration tab would be hidden). If there's no conflict in that respect between creating them via the dashboard and manually importing then this PR is ready to go